Corona-Warn-App/src/test/java/de/rki/coronawarnapp/eventregistration/events/DefaultTraceLocationKtTest.kt

```diff
@@ -13,7 +15,7 @@ internal class DefaultTraceLocationKtTest : BaseTest() {
         TraceLocationEntity(
             guid = "TestGuid",
             version = 1,
-            type = TraceLocation.Type.TEMPORARY_OTHER,
+            type = LOCATION_TYPE_PERMANENT_OTHER,
```

Comment:
Before it was temporary. Was this change on purpose?

Reply:
No, not really on purpose 😅

However, since the only important thing is that the mapper maps the type to the same type, this test is still ok.